Source/Extensions/NSURLRequest.swift

```diff
@@ -10,6 +10,18 @@ public extension NSURLRequest {
     let jsonArray = json as? [AnyObject]
     return jsonArray ?? [:]
   }
+
+  var formStylePayload: [String: String]? {
```

Comment:
`formStylePayload` doesn't seem right ... if we look at the encoding types, you're dealing with `application/x-www-form-urlencoded` right? so maybe `urlEncodedPayload`?